api/routes/recipes.route.js

```diff
@@ -8,4 +8,6 @@ router.get('/', RecipesController.getAllRecipes);
 
 router.post('/', RecipesController.createRecipe);
 
+router.get('/:id', RecipesController.getSingleRecipeFromUrl);
```

Comment:
We're expecting a `:id` but the function is called `FromUrl`, we should make this more consistent